images/router/haproxy/conf/haproxy-config.template

```diff
@@ -32,6 +35,10 @@ global
 defaults
   # maxconn 4096
   # Add x-forwarded-for header.
+{{ if ne (env "ROUTER_SYSLOG_ADDRESS" "") ""}}
+  option httplog
```

Comment:
It won't affect passthrough routes. Since the default logging just logs source and destination addresses and this would do about the same for passthrough routes since there's no http information. 
But that said, it would be a good idea to add `option tcplog` in the backend[s] for passthrough routes. Aka somewhere in this section for the tcp/passthrough backends: 
https://github.com/JacobTanenbaum/origin/blob/7974/images/router/haproxy/conf/haproxy-config.template#L232